proposals/lip-0065.md

```diff
@@ -98,23 +100,31 @@ To summarize, we chose sparse Merkle trees because:
 | Name                                 | Type    | Value                                                 | Description                                                                          |
 |--------------------------------------|---------|-------------------------------------------------------|--------------------------------------------------------------------------------------|
 | `MAX_EVENT_SIZE_BYTES`               | integer | `1024`                                                | Maximum size in bytes of a serialized event.                                         |
+| `MIN_MODULE_NAME_LENGTH`             | integer | `1`                                                   | Minimum length of a module name.                                         |
+| `MAX_MODULE_NAME_LENGTH`             | integer | `32`                                                  | Maximum length of a module name.                                         |
+| `MIN_EVENT_NAME_LENGTH`              | integer | `1`                                                   | Minimum length of an event name.                                         |
+| `MAX_EVENT_NAME_LENGTH`              | integer | `32`                                                  |   Maximum length of an event name.                                         |
 | `EMPTY_HASH`                         | bytes   | `SHA-256("")`                                         | Hash of empty bytes.                                                                 |
 | `EVENT_TOPIC_INIT_GENESIS_STATE`     | bytes   | `0x00`                                                | Default topic for events emitted during genesis state initialization.                |
 | `EVENT_TOPIC_FINALIZE_GENESIS_STATE` | bytes   | `0x01`                                                | Default topic for events emitted during genesis state finalization.                  |
 | `EVENT_TOPIC_BEFORE_TRANSACTIONS`    | bytes   | `0x02`                                                | Default topic for events emitted _before_ transactions execution.                    |
 | `EVENT_TOPIC_AFTER_TRANSACTIONS`     | bytes   | `0x03`                                                | Default topic for events emitted _after_ transactions execution.                     |
-| `EVENT_TYPE_ID_STANDARD`             | bytes   | `0x0000`                                              | Type ID of standard event emitted at the end of transaction execution.               |
+| `EVENT_NAME_STANDARD`                | string   | `executionResult`                                    | Name of standard event emitted at the end of transaction execution.               |
```

Comment:
maybe `transactionResult` since there are other execution steps?

Reply:
I would suggest `CommandExecutionResult` as only the command execution can fail and this event provides the information if this hook failed.

Reply:
If we change the name of this event, I would also not take about "standard events" any more, maybe rename the respective section to "Default Events" and say that by default every module defines the `CommandExecutionResult` event.

Reply:
changed to default